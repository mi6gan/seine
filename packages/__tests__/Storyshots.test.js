import initStoryshots from '@storybook/addon-storyshots';
import { render } from 'enzyme';
import { createSerializer } from 'enzyme-to-json';

initStoryshots({
  renderer: render,
  snapshotSerializers: [createSerializer()],
});
