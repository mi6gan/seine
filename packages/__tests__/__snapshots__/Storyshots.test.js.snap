// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Bream.Review summary header Bad Performance 1`] = `null`;

exports[`Storyshots Bream.Review summary header Good Performance 1`] = `null`;

exports[`Storyshots Bream.Review summary header Great Performance 1`] = `null`;

exports[`Storyshots Content Content Of All Charts 1`] = `null`;

exports[`Storyshots Content Content Of Chart Sample One 1`] = `null`;

exports[`Storyshots Content Content Of Column And Bar Sibling Charts 1`] = `null`;

exports[`Storyshots Content Content Of Column And Line Charts 1`] = `null`;

exports[`Storyshots Content Content Of Line And Bar Sibling Charts 1`] = `null`;

exports[`Storyshots Content Content Of Pie And Bar Sibling Charts 1`] = `null`;

exports[`Storyshots Content Content Of Pie Chart 1`] = `null`;

exports[`Storyshots Content Content Of Six Groups Of Two Columns And Bar Sibling Charts 1`] = `null`;

exports[`Storyshots Content Flex 1`] = `null`;

exports[`Storyshots Content Page 1`] = `null`;

exports[`Storyshots Content Table 1`] = `null`;

exports[`Storyshots Content Text 1`] = `null`;

exports[`Storyshots Editor Editor Of All Charts 1`] = `null`;

exports[`Storyshots Editor Editor Of Chart Sample One 1`] = `null`;

exports[`Storyshots Editor Editor Of Column And Bar Sibling Charts 1`] = `null`;

exports[`Storyshots Editor Editor Of Column And Line Charts 1`] = `null`;

exports[`Storyshots Editor Editor Of Flex 1`] = `null`;

exports[`Storyshots Editor Editor Of Line And Bar Sibling Charts 1`] = `null`;

exports[`Storyshots Editor Editor Of Pie And Bar Sibling Charts 1`] = `null`;

exports[`Storyshots Editor Editor Of Pie Chart 1`] = `null`;

exports[`Storyshots Editor Editor Of Table 1`] = `null`;

exports[`Storyshots Editor Initial Editor 1`] = `null`;
